src/cookbooks/retention.md

```diff
@@ -77,6 +77,23 @@ FROM
 WHERE date = "2019-12-01"
 ```
 
+### Simplified View for 1-week Retention
+
+You can also use the simplified `desktop_retention_1_week` view.  This still supports restriction to an arbitrary population of users by joining against a table containing the `client_id`s of interest.
+
+```sql
+SELECT
+  AVG(IF(retained, 1, 0)) AS retention
+FROM
+  `moz-fx-data-shared-prod.telemetry.desktop_retention_1_week`
+INNER JOIN
```

Comment:
Nit - this might be better as a `RIGHT JOIN` to make sure we're not silently filtering users from `my_cohort`. Not a strong opinion.

Reply:
yeah, that sounds fair